tests/app/dao/test_jobs_dao.py

```diff
@@ -246,8 +247,15 @@ def test_get_scheduled_jobs_gets_ignores_jobs_not_scheduled(notify_db, notify_db
     assert jobs[0].id == job_scheduled.id
 
 
-def test_get_scheduled_jobs_gets_ignores_jobs_scheduled_in_the_future(notify_db, notify_db_session):
-    one_minute_in_the_future = datetime.utcnow() + timedelta(minutes=1)
-    sample_job(notify_db, notify_db_session, scheduled_for=one_minute_in_the_future, job_status='scheduled')
+def test_get_scheduled_jobs_gets_ignores_jobs_scheduled_in_the_future(
```

Comment:
1. Fixed in e28ce50
2. There’s already a test for this here: https://github.com/alphagov/notifications-api/blob/add-cancel-job-endpoint/tests/app/dao/test_jobs_dao.py#L230-L238